database/migrations/2017_03_14_024733_UpdateSharedExpensesTable.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class UpdateSharedExpensesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('sharedexpenses', function (Blueprint $table) {
+            // Primary keys
+            $table->primary('expense_id', 'secondary_username');
```

Comment:
You can make primary key ids :
$table->increments('id');
which is like saying 
id UNSIGNED INT PRIMARY KEY AUTO INCREMENT